Makefile

```diff
@@ -8,7 +8,7 @@ ORCHESTRATOR = docker
 
 .PHONY: help
 help:
-	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort -k 1,1 | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' Makefile | sort -k 1,1 | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
```

Comment:
y tho?

Reply:
if you haz a `Makefile.local` the names would get lost due to how grep works
![image](https://github.com/user-attachments/assets/54646712-91b1-4c79-a6e4-eb48d7df34e3)